packages/flutter/lib/src/widgets/_html_element_view_web.dart

```diff
@@ -120,10 +129,25 @@ class _HtmlElementViewController extends PlatformViewController {
     // cross-origin iframes, which only accept user-generated events.
   }
 
+  @override
+  Future<void> moveToFlutterView(int flutterViewId) async {
+    if (_initialized) {
+      final Map<String, Object?> args = <String, Object?>{
+        'platformViewId': platformViewId,
+        'viewId': flutterViewId,
+      };
+      await SystemChannels.platform_views.invokeMethod<void>('moveToFlutterView', args);
```

Comment:
```suggestion
      await SystemChannels.platform_views.invokeMethod<void>('move', args);
```

👍 I'd call this `move` just to keep the same brevity of the `create`/`dispose` methods, but I don't have a strong opinion.